.cirrus.yml

```diff
@@ -1,547 +1,498 @@
-container:
-  image: gcr.io/flutter-cirrus/build-flutter-image:latest
+# CIRRUS CONFIGURATION FILE
 
+# We change Flutter's directory to include a space in its name (see
+# $CIRRUS_WORKING_DIR) so that we constantly test path names with
+# spaces in them.
+
+env:
+  GCLOUD_SERVICE_ACCOUNT_KEY: ENCRYPTED[f12abe60f5045d619ef4c79b83dd1e0722a0b0b13dbea95fbe334e2db7fffbcd841a5a92da8824848b539a19afe0c9fb]
+  #BOT: true
```

Comment:
Remove

Reply:
The comment was temporary, I've already put it back. I was testing the theory that this was causing the Windows issue.